diff --git a/node_modules/react-native-paper/lib/typescript/types.d.ts b/node_modules/react-native-paper/lib/typescript/types.d.ts
index cc0320a..e6e6488 100644
--- a/node_modules/react-native-paper/lib/typescript/types.d.ts
+++ b/node_modules/react-native-paper/lib/typescript/types.d.ts
@@ -1,5 +1,6 @@
 import type * as React from 'react';
 import type { $DeepPartial } from '@callstack/react-theme-provider';
+import type { Theme as NativeNavigationTheme } from '@react-navigation/native';
 export declare type Font = {
     fontFamily: string;
     fontWeight?: 'normal' | 'bold' | '100' | '200' | '300' | '400' | '500' | '600' | '700' | '800' | '900';
@@ -161,16 +162,6 @@ export declare type MD3ElevationColors = {
 export declare type $Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
 export declare type $RemoveChildren<T extends React.ComponentType<any>> = $Omit<React.ComponentPropsWithoutRef<T>, 'children'>;
 export declare type EllipsizeProp = 'head' | 'middle' | 'tail' | 'clip';
-export declare type NavigationTheme = {
-    dark: boolean;
-    colors: {
-        primary: string;
-        background: string;
-        card: string;
-        text: string;
-        border: string;
-        notification: string;
-    };
-};
+export declare type NavigationTheme = NativeNavigationTheme;
 export {};
 //# sourceMappingURL=types.d.ts.map
diff --git a/node_modules/react-native-paper/src/components/BottomNavigation/BottomNavigationBar.tsx b/node_modules/react-native-paper/src/components/BottomNavigation/BottomNavigationBar.tsx
index 0bfe303..a59ae06 100644
--- a/node_modules/react-native-paper/src/components/BottomNavigation/BottomNavigationBar.tsx
+++ b/node_modules/react-native-paper/src/components/BottomNavigation/BottomNavigationBar.tsx
@@ -360,7 +360,7 @@ const BottomNavigationBar = <Route extends BaseRoute>({
   navigationState,
   renderIcon,
   renderLabel,
-  renderTouchable = (props: TouchableProps<Route>) => <Touchable {...props} />,
+  renderTouchable = ({ key, ...props }: TouchableProps<Route>) => <Touchable key={key} {...props} />,
   getLabelText = ({ route }: { route: Route }) => route.title,
   getBadge = ({ route }: { route: Route }) => route.badge,
   getColor = ({ route }: { route: Route }) => route.color,
diff --git a/node_modules/react-native-paper/src/types.tsx b/node_modules/react-native-paper/src/types.tsx
index 175131c..c7961b8 100644
--- a/node_modules/react-native-paper/src/types.tsx
+++ b/node_modules/react-native-paper/src/types.tsx
@@ -203,14 +203,4 @@ export type $RemoveChildren<T extends React.ComponentType<any>> = $Omit<
 
 export type EllipsizeProp = 'head' | 'middle' | 'tail' | 'clip';
 
-export type NavigationTheme = {
-  dark: boolean;
-  colors: {
-    primary: string;
-    background: string;
-    card: string;
-    text: string;
-    border: string;
-    notification: string;
-  };
-};
+export type { Theme as NavigationTheme } from '@react-navigation/native';
