diff --git a/node_modules/react-native-paper-dates/src/Date/Swiper.tsx b/node_modules/react-native-paper-dates/src/Date/Swiper.tsx
index 08c47c1..bdd49b5 100644
--- a/node_modules/react-native-paper-dates/src/Date/Swiper.tsx
+++ b/node_modules/react-native-paper-dates/src/Date/Swiper.tsx
@@ -21,6 +21,7 @@ import React, {
 import { sharedStyles } from '../shared/styles'
 
 const useIsomorphicLayoutEffect =
+// @ts-ignore
   typeof window !== 'undefined' ? useLayoutEffect : useEffect
 
 function Swiper({
@@ -120,6 +121,7 @@ function VerticalScroller({
     const top =
       getVerticalMonthsOffset(idx.current, startWeekOnMonday) - montHeaderHeight
 
+    // @ts-ignore
     element.scrollTo({
       top,
     })
@@ -129,6 +131,7 @@ function VerticalScroller({
 
   const onScroll = useCallback(
     (e: UIEvent) => {
+      // @ts-ignore
       const top = e.currentTarget?.scrollTop
       if (top === 0) {
         return
@@ -206,6 +209,7 @@ export function useDebouncedCallback(callback: any): any {
     return () => {
       mounted.current = false
       if (timerId.current) {
+        // @ts-ignore
         window.cancelAnimationFrame(timerId.current)
       }
     }
@@ -214,8 +218,10 @@ export function useDebouncedCallback(callback: any): any {
   return useCallback(
     (args: any) => {
       if (timerId.current) {
+        // @ts-ignore
         window.cancelAnimationFrame(timerId.current)
       }
+      // @ts-ignore
       timerId.current = window.requestAnimationFrame(function () {
         if (mounted.current) {
           latest.current(args)
