diff --git a/node_modules/react-native-paper-dates/src/Date/Swiper.tsx b/node_modules/react-native-paper-dates/src/Date/Swiper.tsx
index a71e0f2..c455c37 100644
--- a/node_modules/react-native-paper-dates/src/Date/Swiper.tsx
+++ b/node_modules/react-native-paper-dates/src/Date/Swiper.tsx
@@ -32,6 +32,7 @@ import {
 import { sharedStyles } from '../shared/styles'
 
 const useIsomorphicLayoutEffect =
+  // @ts-ignore
   typeof window !== 'undefined' ? useLayoutEffect : useEffect
 
 function Swiper({
@@ -177,6 +178,7 @@ function VerticalScroller({
         effectiveEndYear
       ) - montHeaderHeight
 
+    // @ts-ignore
     element.scrollTo({
       top,
     })
@@ -186,6 +188,7 @@ function VerticalScroller({
 
   const onScroll = useCallback(
     (e: UIEvent) => {
+      // @ts-ignore
       const top = e.currentTarget?.scrollTop
       if (top === 0) {
         return
@@ -298,6 +301,7 @@ export function useDebouncedCallback(callback: any): any {
     return () => {
       mounted.current = false
       if (timerId.current) {
+        // @ts-ignore
         window.cancelAnimationFrame(timerId.current)
       }
     }
@@ -306,8 +310,10 @@ export function useDebouncedCallback(callback: any): any {
   return useCallback(
     (args: any) => {
       if (timerId.current) {
+        // @ts-ignore
         window.cancelAnimationFrame(timerId.current)
       }
+      // @ts-ignore
       timerId.current = window.requestAnimationFrame(function () {
         if (mounted.current) {
           latest.current(args)
diff --git a/node_modules/react-native-paper-dates/src/shared/utils.ts b/node_modules/react-native-paper-dates/src/shared/utils.ts
index 831129b..32ec4f9 100644
--- a/node_modules/react-native-paper-dates/src/shared/utils.ts
+++ b/node_modules/react-native-paper-dates/src/shared/utils.ts
@@ -5,7 +5,7 @@ import {
   overlay,
   useTheme,
 } from 'react-native-paper'
-import Color from 'color'
+import Color, { ColorInstance } from 'color';
 
 export const supportedOrientations: (
   | 'portrait'
@@ -78,12 +78,12 @@ export function range(start: number, end: number) {
     .map((_, i) => start + i)
 }
 
-export function lightenBy(color: Color, ratio: number) {
+export function lightenBy(color: ColorInstance, ratio: number) {
   const lightness = color.lightness()
   return color.lightness(lightness + (100 - lightness) * ratio)
 }
 
-export function darkenBy(color: Color, ratio: number) {
+export function darkenBy(color: ColorInstance, ratio: number) {
   const lightness = color.lightness()
   return color.lightness(lightness - lightness * ratio)
 }
